strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/booters/StorageBooter.java

```diff
@@ -93,26 +96,29 @@ public void removeLock()
         logger.debug("Removed lock file '" + lockFile.getAbsolutePath() + "'.");
     }
 
-    public void createTempDir()
+    public void createTempDir() throws IOException
     {
-        File tempDir = new File(ConfigurationResourceResolver.getVaultDirectory(), "tmp");
-        if (!tempDir.exists())
+        String tempDirLocation = System.getProperty("java.io.tmpdir",
+                                                Paths.get(ConfigurationResourceResolver.getVaultDirectory(), "tmp")
```

Comment:
This should be `.temp`.

Reply:
hm.... why?
this is not related with concrete repository, if I understand you correctly

Reply:
Sorry, my bad! Seems I got confused.